Package.swift

```diff
@@ -33,7 +33,7 @@ let package = Package(
       url: "https://github.com/apple/swift-tools-support-core.git",
       .branch("release/5.6")
     ),
-    .package(url: "https://github.com/vapor/vapor.git", from: "4.53.0"),
+    .package(url: "https://github.com/vapor/vapor.git", from: "4.57.1"),
```

Comment:
This update is required to include https://github.com/vapor/vapor/pull/2823, which allows us to serve `.mjs` files with correct `Content-Type` header.